src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1541,7 +1546,17 @@ internal virtual async Task OnSourceFileAdded(SessionId sessionId, SourceFile so
             {
                 if (req.TryResolve(source))
                 {
-                    await SetBreakpoint(sessionId, context.store, req, true, false, token);
+                    try
+                    {
+                        await SetBreakpoint(sessionId, context.store, req, true, false, token);
+                    }
+                    catch (DebuggerAgentException e)
+                    {
+                        //it's not a wasm page then the command throws an error
+                        if (!e.Message.Contains("getDotnetRuntime is not defined"))
```

Comment:
Might be useful to add a comment next to `getDotnetRuntime` saying that when changing the name, instances in debugger should be changed too. And we might have this in more than one place?